search-resources-api/src/test/java/RequestUtilTest.java

```diff
@@ -0,0 +1,21 @@
+import no.unit.nva.search.RequestUtil;
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.handlers.RequestInfo;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RequestUtilTest {
+
+    public static final String SAMPLE_QUERY_PARAMETER = "SampleQueryParameter";
+
+    @Test
+    public void canSearchTermRequest() throws ApiGatewayException {
```

Comment:
Is this not tested by handler testing?

Reply:
removed test as it was created for code coverage a long time ago (and test didn't d anythin useful anyway..)